pyta.py

```diff
@@ -30,6 +30,9 @@ def check(module_name):
     linter = lint.PyLinter(reporter=reporter)
     linter.load_default_plugins()
     linter.load_plugin_modules(['checkers'])
+    linter.load_plugin_modules(['global_variables_checker'])
+    linter.read_config_file()
```

Comment:
I've tried that but there are some conflicts arising and I'm not sure where I can see what the conflicts are.